custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -0,0 +1,236 @@
+{
+    "meta": { 
+        "label": "Ontology tagging",
+        "description": "Tag documents matching keywords within a corpus of text documents",
+        "icon": "icon-pushpin"
+    },
+    "kind": "PYTHON",
+    "inputRoles": [
+        {
+            "name": "Document dataset",
```

Comment:
As a convention, it's better to use names in_this_case, without spaces. This is because names are used as variable names and keys in the python backend.

Reply:
in 80b523a